jsonnet/thanos-mixin/alerts/store.libsonnet

```diff
@@ -0,0 +1,88 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'thanos-store.rules',
+        rules: [
+          {
+            alert: 'ThanosStoreGrpcErrorRate',
```

Comment:
It's a generic component metric. We can reuse this alert for all Thanos components.